appkeeper/assets/dashboards/appkeeper_dashboard.json

```diff
@@ -0,0 +1,123 @@
+{
+  "title": "AppKeeper dashboard",
```

Comment:
Can we change the dashboard to instead be a screen board? I also recommend following the dashboard best practices found here:

https://datadoghq.dev/integrations-core/guidelines/dashboards/